test/extensions/filters/http/alternate_protocols_cache/filter_integration_test.cc

```diff
@@ -179,10 +193,28 @@ class MixedUpstreamIntegrationTest : public FilterIntegrationTest {
   bool use_http2_{false};
 };
 
+int getRtt(std::string alt_svc, TimeSource& time_source) {
+  auto data = Http::AlternateProtocolsCacheImpl::originDataFromString(alt_svc, time_source);
+  return data.has_value() ? data.value().srtt.count() : 0;
+}
 // Test auto-config with a pre-populated HTTP/3 alt-svc entry. The upstream request will
 // occur over HTTP/3.
 TEST_P(MixedUpstreamIntegrationTest, BasicRequestAutoWithHttp3) {
   testRouterRequestAndResponseWithBody(0, 0, false);
+  cleanupUpstreamAndDownstream();
+  std::string alt_svc;
+
+  // Make sure the srtt gets updated to a non-zero value.
+  for (int i = 0; i < 5; ++i) {
+    // Make sure that srtt is updated.
+    const std::string filename = TestEnvironment::temporaryPath("alt_svc_cache.txt");
+    alt_svc = TestEnvironment::readFileToStringForTest(filename);
+    if (getRtt(alt_svc, timeSystem()) != 0) {
+      break;
+    }
+    timeSystem().advanceTimeWait(std::chrono::seconds(1));
```

Comment:
OOC, does this wait for a wall-clock 1 second, or is this using some sort of mock time?